www/js/models/form.js

```diff
@@ -60,6 +60,18 @@ FormModel = Backbone.Model.extend({
                 sanitizeField(form.fields[field]);
             }
         }
+
+        // loop over and sanitize pages in the form
+        for (var i = 0; i < form.pages.length; i++) {
```

Comment:
Is it ever possible that a form without any pages is displayed? Or is at least 1 page always given?

Reply:
That is a good question. From that i have seen a form always has at least one page but could be worth putting a check in just in case.

Reply:
@pb82 I'v pushed up a change to instantiate the pages as form.pages or an empty array. To avoid an undefined error.